package.json

```diff
@@ -30,15 +30,14 @@
     "@ory/elements-markup": "0.0.1-beta.7",
     "@ory/integrations": "1.1.4",
     "axios": "1.2.6",
-    "body-parser": "1.20.1",
+    "body-parser": "1.20.2",
     "cookie-parser": "1.4.6",
     "csurf": "1.11.0",
     "express": "4.18.2",
     "express-handlebars": "6.0.7",
-    "express-jwt": "8.4.0",
-    "express-winston": "4.2.0",
+    "express-jwt": "8.4.1",
     "jwks-rsa": "3.0.1",
-    "winston": "3.8.2"
+    "express-winston": "4.2.0"
```

Comment:
Why does the import not need to change?

Reply:
https://github.com/Marak/colors.js/issues/285

Reply:
afaik, winston is not needed explicitly since we import `express-winston`

Reply:
Ahh, yes, makes sense...

Reply:
i'll update it